File: Problem_Solving/stiver-a2z-sheet/05-basic-recursion/sumOfSeries.js
class Solution {
  /**
* @param number n

* @returns number
*/
  sumOfSeries(n) {
    // code here

    // return Math.pow((n * (n + 1) /2) ,2)
    let bigN = BigInt(n);
    return (((bigN * (bigN + 1n)) / 2n) ** 2n).toString();
  }
}

File: Problem_Solving/stiver-a2z-sheet/05-basic-recursion/sumOfSeries.py
class Solution:
    def sumOfSeries(self,n):
        # #code here
        # if n == 1:
        #     return 1
        # return n**3 + self.sumOfSeries(n -1)

        # sum = 0
        # for i in range(1, n+1):
        #     sum += i**3
        # return sum

        return (n * (n + 1)//2) **2
